paparazzi/src/main/java/app/cash/paparazzi/Paparazzi.kt

```diff
@@ -625,9 +626,12 @@ class Paparazzi @JvmOverloads constructor(
 
   private fun DeviceConfig.updateIfAccessibilityTest(): DeviceConfig =
     if (renderExtensions.any { it is AccessibilityRenderExtension }) {
+      val newWidth = screenWidth * 2
+      val newOrientation = if (newWidth > screenHeight) ScreenOrientation.LANDSCAPE else ScreenOrientation.PORTRAIT
```

Comment:
wait, why is this still necessary?

Reply:
If we don't set the orientation then the `DeviceConfig` will flip the orientation, resulting in a taller screen because it will use the new 'width' as a height value instead

Reply:
got it.  hmm, ok, let's leave this as is then and figure out a better long-term solution.